tests/translator/test_translator.py

```diff
@@ -259,6 +259,7 @@ class TestTranslatorEndToEnd(TestCase):
         'api_with_apikey_default_override',
         'api_with_apikey_required',
         'api_with_path_parameters',
+        'kinesis_stream'
```

Comment:
Please follow the naming convention "function_with_batch_window...."

Reply:
renamed 'kinesis_stream' with 'function_with_batch_window'